internal_transfers/actions/scripts/backfill.ts

```diff
@@ -0,0 +1,60 @@
+import { backFillTokenImbalances } from "./lib";
+import { EnsoSimulator } from "../src/simulate/enso";
+import { TenderlySimulator } from "../src/simulate/tenderly";
+import { TransactionSimulator } from "../src/simulate/interface";
+import { parse } from "ts-command-line-args";
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+interface RuntimeArgs {
+  // YYYY-MM-DD
+  dateFrom: string;
+  dateTo: string;
+}
+
+const {
+  DUNE_API_KEY,
+  DB_URL,
+  NODE_URL,
+  TENDERLY_ACCESS_KEY,
+  TENDERLY_USER,
+  TENDERLY_PROJECT,
+} = process.env;
+
+if (!DUNE_API_KEY) {
+  throw new Error("Missing DUNE_API_KEY");
+}
+if (!DB_URL) {
+  throw new Error("Missing DB_URL");
+}
+if (!NODE_URL) {
+  throw new Error("Missing NODE_URL");
+}
+
+let simulator: TransactionSimulator = new EnsoSimulator(
+  "http://127.0.0.1:8080/api/v1/simulate"
+);
+if (
+  ![TENDERLY_ACCESS_KEY, TENDERLY_USER, TENDERLY_PROJECT].includes(undefined)
+) {
+  console.log("Using Tenderly Simulator");
+  simulator = new TenderlySimulator(
+    TENDERLY_USER!,
+    TENDERLY_PROJECT!,
+    TENDERLY_ACCESS_KEY!
+  );
+}
+
+export const args = parse<RuntimeArgs>({
+  dateFrom: String,
+  dateTo: String,
+});
```

Comment:
Nitpick: do you care about validating these dates match the format you want?

Reply:
You could do everyone's favorite pattern matching tool: regex!
Or a validation lib like https://www.npmjs.com/package/env-schema but I think that's overkill

Reply:
https://github.com/cowprotocol/solver-rewards/issues/289